<a href="https://colab.research.google.com/github/khansan111/Video-games/blob/main/AdidasUsa_Sales.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
import pandas as pd

In [2]:
from google.colab import files

In [3]:
uploeded = files.upload()

Saving adidas_usa.csv to adidas_usa.csv


In [4]:
df = pd.read_csv('adidas_usa.csv')

In [5]:
print (df.head)

<bound method NDFrame.head of      index                                                url  \
0        0  https://www.adidas.com/us/beach-shorts/FJ5089....   
1        1  https://www.adidas.com/us/five-ten-kestrel-lac...   
2        2  https://www.adidas.com/us/mexico-away-jersey/G...   
3        3  https://www.adidas.com/us/five-ten-hiangle-pro...   
4        4  https://www.adidas.com/us/mesh-broken-stripe-p...   
..     ...                                                ...   
840    840  https://www.adidas.com/us/supernova-plus-shoes...   
841    841  https://www.adidas.com/us/choigo-shoes/H00667....   
842    842  https://www.adidas.com/us/daily-3.0-shoes/GZ77...   
843    843  https://www.adidas.com/us/daily-3.0-shoes/GZ77...   
844    844  https://www.adidas.com/us/choigo-shoes/FY6503....   

                                                name     sku  selling_price  \
0                                       Beach Shorts  FJ5089             40   
1          Five Ten Kestrel Lac

# **summary statistics**

In [6]:
print(df.describe())

            index  selling_price  average_rating  reviews_count
count  845.000000     845.000000      845.000000     845.000000
mean   422.000000      53.192899        4.608402     426.178698
std    244.074784      31.411645        0.293795    1229.158277
min      0.000000       9.000000        1.000000       1.000000
25%    211.000000      28.000000        4.500000      19.000000
50%    422.000000      48.000000        4.700000      68.000000
75%    633.000000      70.000000        4.800000     314.000000
max    844.000000     240.000000        5.000000   11750.000000


* Count: The number of non-null entries for each column. Here, all columns have 845 entries.

* Mean: The average value for each column. For example, the average selling price is approximately $53.19.

* Std (standard deviation): This indicates the variability or dispersion of the data. A higher standard deviation means the data points are spread out over a wider range. For example, the selling prices have a standard deviation of about $31.41.

* min: The minimum value in each column. The lowest selling price is $9.

* 25% (1st quartile): This value indicates that 25% of the data points fall below this value. For example, 25% of products have a selling price of $28 or lower.

* 50% (median): This is the middle value when the data is sorted. It shows that half the products have a selling price below $48.

* 75% (3rd quartile): This indicates that 75% of the data points are below this value. For selling prices, 75% of products are priced at $70 or lower.

* Max: The maximum value in each column. The highest selling price is $240.

In [7]:
print(df.isnull().sum())

index              0
url                0
name               0
sku                0
selling_price      0
original_price    16
currency           0
availability       0
color              0
category           0
source             0
source_website     0
breadcrumbs        0
description        0
brand              0
images             0
country            0
language           0
average_rating     0
reviews_count      0
crawled_at         0
dtype: int64


# **Cheapest item**

In [20]:
# Find the cheapest item in the entire DataFrame
cheapest_item = df.nsmallest(1, 'selling_price')

# Extract relevant details
cheapest_price = cheapest_item['selling_price'].values[0]
original_price = cheapest_item['original_price'].values[0]
item_name = cheapest_item['name'].values[0]
sku = cheapest_item['sku'].values[0]

print(f"The cheapest item is '{item_name}' (SKU: {sku}) with a selling price of ${cheapest_price} and an original price of ${original_price}.")


The cheapest item is 'Fold-Cuff Crew Socks 3 Pairs' (SKU: GN4894) with a selling price of $9 and an original price of $$14.


In [15]:
# Find the cheapest item in the entire DataFrame
cheapest_item = df.nsmallest(1, 'selling_price')

# Extract relevant details
cheapest_price = cheapest_item['selling_price'].values[0]
original_price = cheapest_item['original_price'].values[0]
item_name = cheapest_item['name'].values[0]
sku = cheapest_item['sku'].values[0]

# Convert original_price to float, removing dollar signs and commas if present
original_price = float(original_price.replace('$', '').replace(',', '')) # Only apply replace to original_price

# Calculate loss
loss = original_price - cheapest_price

# Print item details
print(f"The cheapest item is '{item_name}' (SKU: {sku}) with a selling price of ${cheapest_price:.2f} and an original price of ${original_price:.2f}.")

# Display loss information
if loss > 0:
    print(f"Loss from this item: ${loss:.2f}")
else:
    print(f"No loss from this item; it's either break-even or profitable.")

The cheapest item is 'Fold-Cuff Crew Socks 3 Pairs' (SKU: GN4894) with a selling price of $9.00 and an original price of $14.00.
Loss from this item: $5.00


# The most expensive sold **item**

In [21]:
# Find the most expensive item in the entire DataFrame
most_expensive_item = df.nlargest(1, 'selling_price')

# Extract relevant details
most_expensive_price = most_expensive_item['selling_price'].values[0]
original_price = most_expensive_item['original_price'].values[0]
item_name = most_expensive_item['name'].values[0]
sku = most_expensive_item['sku'].values[0]

print(f"The most expensive sold item is '{item_name}' (SKU: {sku}) with a selling price of ${most_expensive_price} and an original price of ${original_price}.")


The most expensive sold item is 'Terrex Primeknit Rain.RDY Jacket' (SKU: GI7309) with a selling price of $240 and an original price of $$300.


In [23]:
# Extract additional details
brand = most_expensive_item['brand'].values[0]
category = most_expensive_item['category'].values[0]
average_rating = most_expensive_item['average_rating'].values[0]
reviews_count = most_expensive_item['reviews_count'].values[0]

# Display additional information
print(f"Brand: {brand}")
print(f"Category: {category}")
print(f"Average Rating: {average_rating} ({reviews_count} reviews)")
print(f"Original Price: ${original_price}")


Brand: adidas
Category: Clothing
Average Rating: 5.0 (4 reviews)
Original Price: $$300


In [9]:
# Step 2: Find the most expensive item in the DataFrame
most_expensive_item = df.nlargest(1, 'selling_price')

# Step 3: Extract relevant details
most_expensive_price = most_expensive_item['selling_price'].values[0]
original_price = most_expensive_item['original_price'].values[0]
brand = most_expensive_item['brand'].values[0]
category = most_expensive_item['category'].values[0]
average_rating = most_expensive_item['average_rating'].values[0]
reviews_count = most_expensive_item['reviews_count'].values[0]

# Remove dollar signs and convert prices to float
# The original_price likely contains a dollar sign, so we only need to replace on this variable
original_price = float(original_price.replace('$', '').replace(',', ''))

# most_expensive_price is already an integer, so we can cast it to a float if needed
most_expensive_price = float(most_expensive_price)

# Display additional information
print(f"Brand: {brand}")
print(f"Category: {category}")
print(f"Average Rating: {average_rating} ({reviews_count} reviews)")
print(f"Original Price: ${original_price:.2f}")

# Calculate profit
profit = most_expensive_price - original_price

# Display profit information
if profit > 0:
    print(f"Profit from this item: ${profit:.2f}")
else:
    print(f"Loss from this item: ${-profit:.2f}")

Brand: adidas
Category: Clothing
Average Rating: 5.0 (4 reviews)
Original Price: $300.00
Loss from this item: $60.00


# **The most sold item**

In [11]:
# Find the most sold item based on reviews count
most_sold_item = df.nlargest(1, 'reviews_count')

# Extract relevant details
item_name = most_sold_item['name'].values[0]
reviews_count = most_sold_item['reviews_count'].values[0]
selling_price = most_sold_item['selling_price'].values[0]
sku = most_sold_item['sku'].values[0]

print(f"The most sold item is '{item_name}' (SKU: {sku}) with {reviews_count} reviews, selling at ${selling_price}.")


The most sold item is 'Stan Smith Shoes' (SKU: GZ7767) with 11750 reviews, selling at $72.


In [13]:
# Find the most sold item based on reviews count
most_sold_item = df.nlargest(1, 'reviews_count')

# Extract relevant details
item_name = most_sold_item['name'].values[0]
reviews_count = most_sold_item['reviews_count'].values[0]
selling_price = most_sold_item['selling_price'].values[0]
original_price = most_sold_item['original_price'].values[0]  # Extract original price
sku = most_sold_item['sku'].values[0]

# Convert original_price to float after removing dollar sign
# If original_price is already numeric, just cast to float
try:
    original_price = float(original_price.replace('$', '').replace(',', ''))
except AttributeError:
    original_price = float(original_price)

# selling_price is likely already numeric, so just cast to float
selling_price = float(selling_price)

# Calculate profit
profit = selling_price - original_price

# Print item details
print(f"The most sold item is '{item_name}' (SKU: {sku}) with {reviews_count} reviews, selling at ${selling_price:.2f}.")

# Display profit information
if profit > 0:
    print(f"Profit from this item: ${profit:.2f}")
else:
    print(f"Loss from this item: ${-profit:.2f}")


The most sold item is 'Stan Smith Shoes' (SKU: GZ7767) with 11750 reviews, selling at $72.00.
Loss from this item: $18.00


# **The most review item**

In [11]:
# Find the most sold item based on reviews count
most_sold_item = df.nlargest(1, 'reviews_count')

# Extract relevant details
item_name = most_sold_item['name'].values[0]
reviews_count = most_sold_item['reviews_count'].values[0]
selling_price = most_sold_item['selling_price'].values[0]
original_price = most_sold_item['original_price'].values[0]  # Extract original price
sku = most_sold_item['sku'].values[0]

# Convert original_price to float after removing dollar sign
# If original_price is already numeric, just cast to float
try:
    # original_price is a string, so we can use replace
    original_price = float(original_price.replace('$', '').replace(',', ''))
except AttributeError:
    # original_price is not a string (likely already numeric), so just cast to float
    original_price = float(original_price)

# selling_price is likely already numeric, so just cast to float
selling_price = float(selling_price)

# Calculate profit
profit = selling_price - original_price

# Print item details
print(f"The most sold item is '{item_name}' (SKU: {sku}) with {reviews_count} reviews, selling at ${selling_price:.2f}.")

# Display profit information
if profit > 0:
    print(f"Profit from this item: ${profit:.2f}")
else:
    print(f"Loss from this item: ${-profit:.2f}")

The most sold item is 'Stan Smith Shoes' (SKU: GZ7767) with 11750 reviews, selling at $72.00.
Loss from this item: $18.00


# **The Least Review**

In [17]:
# Find the item with the least reviews in the entire DataFrame
least_reviewed_item = df.nsmallest(1, 'reviews_count')

# Extract relevant details
item_name = least_reviewed_item['name'].values[0]
sku = least_reviewed_item['sku'].values[0]
reviews_count = least_reviewed_item['reviews_count'].values[0]
selling_price = least_reviewed_item['selling_price'].values[0]
original_price = least_reviewed_item['original_price'].values[0]
average_rating = least_reviewed_item['average_rating'].values[0]

# Convert prices to float, handling potential dollar signs and commas for original_price
# selling_price is likely numeric, so directly cast to float
selling_price = float(selling_price)

try:
    # original_price might be a string, attempt to remove '$' and ',' before converting to float
    original_price = float(original_price.replace('$', '').replace(',', ''))
except AttributeError:
    # If original_price is already numeric, directly cast to float
    original_price = float(original_price)

# Print item details
print(f"The item with the least reviews is '{item_name}' (SKU: {sku}) with only {reviews_count} reviews, selling at ${selling_price:.2f} and an origin")

The item with the least reviews is 'Crew Sweatshirt' (SKU: HG6687) with only 1 reviews, selling at $60.00 and an origin


**Can we identify customer favorite shoes color** based on  the most selling shoes color to know which color of shoes that need to be produce more

In [12]:
# Find the item with the most reviews in the entire DataFrame
most_reviewed_item = df.nlargest(1, 'reviews_count')

# Extract relevant details
item_name = most_reviewed_item['name'].values[0]
sku = most_reviewed_item['sku'].values[0]
color = most_reviewed_item['color'].values[0]
reviews_count = most_reviewed_item['reviews_count'].values[0]
average_rating = most_reviewed_item['average_rating'].values[0]

# Print item details
print(f"The most favorite shoe color is '{color}' for the item '{item_name}' (SKU: {sku}) with {reviews_count} reviews.")
print(f"Average Rating: {average_rating}")


The most favorite shoe color is 'White' for the item 'Stan Smith Shoes' (SKU: GZ7767) with 11750 reviews.
Average Rating: 4.7


# **A/B Testing Analysis**

In [21]:
# Sample data for demonstration
data = {
    'category': ['Sneakers', 'Sneakers', 'Boots', 'Boots', 'Sneakers', 'Boots', 'Sandals', 'Sneakers', 'Sandals', 'Boots'],
    'sales': [100, 150, 200, 250, 180, 220, 130, 160, 140, 260],
    'reviews_count': [10, 15, 25, 30, 20, 22, 8, 12, 10, 35],
    'average_rating': [4.5, 4.6, 4.3, 4.8, 4.5, 4.4, 4.0, 4.7, 4.5, 4.2]
}

# Create a DataFrame
df_sales = pd.DataFrame(data)

# Analyze sales data by category
results = df_sales.groupby('category').agg({
    'sales': 'sum',
    'reviews_count': 'sum',
    'average_rating': 'mean'
}).reset_index()

# Display the results
print("A/B Testing Results by Shoe Category:")
print(results)


A/B Testing Results by Shoe Category:
   category  sales  reviews_count  average_rating
0     Boots    930            112           4.425
1   Sandals    270             18           4.250
2  Sneakers    590             57           4.575


# **A/B testing Sneaker Sales**

In [22]:
# Sample data for sneaker sales analysis
data = {
    'campaign': ['Campaign A', 'Campaign A', 'Campaign B', 'Campaign B', 'Campaign A', 'Campaign B', 'Campaign A'],
    'sales': [100, 150, 200, 250, 180, 220, 160],
    'reviews_count': [10, 15, 20, 30, 12, 22, 18],
    'average_rating': [4.5, 4.6, 4.3, 4.8, 4.5, 4.4, 4.6]
}

# Create a DataFrame
df_sales = pd.DataFrame(data)

# Analyze sneaker sales data by campaign
results = df_sales.groupby('campaign').agg({
    'sales': 'sum',
    'reviews_count': 'sum',
    'average_rating': 'mean'
}).reset_index()

# Display the results
print("A/B Testing Results for Sneaker Sales:")
print(results)

A/B Testing Results for Sneaker Sales:
     campaign  sales  reviews_count  average_rating
0  Campaign A    590             55            4.55
1  Campaign B    670             72            4.50


Campaign B was more successful in terms of sales and customer engagement, Campaign A provided slightly better customer satisfaction. By analyzing these results and implementing the recommendations, future marketing strategies can be optimized to maximize both sales and customer satisfaction.

# **A/B testing on Boots**

In [25]:
import pandas as pd

# Sample data for boot sales analysis
data = {
    'campaign': ['Campaign A', 'Campaign A', 'Campaign B', 'Campaign B', 'Campaign A', 'Campaign B', 'Campaign A'],
    'sales': [120, 180, 220, 300, 200, 250, 150],  # Sample sales figures for boots
    'reviews_count': [15, 20, 25, 35, 18, 30, 22],  # Sample review counts for boots
    'average_rating': [4.4, 4.5, 4.2, 4.7, 4.6, 4.3, 4.5]  # Sample average ratings for boots
}

# Create a DataFrame
df_sales = pd.DataFrame(data)

# Analyze boot sales data by campaign
results = df_sales.groupby('campaign').agg({
    'sales': 'sum',              # Total sales per campaign
    'reviews_count': 'sum',      # Total reviews per campaign
    'average_rating': 'mean'     # Average rating per campaign
}).reset_index()

# Display the results
print("A/B Testing Results for Boot Sales:")
print(results)


A/B Testing Results for Boot Sales:
     campaign  sales  reviews_count  average_rating
0  Campaign A    650             75             4.5
1  Campaign B    770             90             4.4


**Campaign Performance:**

* Campaign B significantly outperformed Campaign A in total sales and reviews, making it the more effective marketing strategy in this analysis.
Customer Engagement:

* The increase in reviews for Campaign B suggests a higher level of customer engagement, which is vital for brand loyalty and future sales.
Customer Satisfaction:

* Although Campaign B had lower average ratings, it is crucial to investigate the reasons behind this to ensure that customer satisfaction is not compromised in the pursuit of sales.
Recommendations:

Future Campaigns: Consider using a combination of successful elements from both campaigns, such as the BOGO strategy with additional incentives to enhance customer satisfaction.
Customer Feedback: Collect qualitative feedback from customers to understand their experiences and address any issues affecting the average rating.
Continuous Monitoring: Implement ongoing A/B testing and monitoring of customer feedback to adapt and improve future marketing strategies.

# **A/B testing on sandals sales**

In [26]:
import pandas as pd

# Sample data for sandals sales analysis
data = {
    'campaign': ['Campaign A', 'Campaign A', 'Campaign B', 'Campaign B', 'Campaign A', 'Campaign B', 'Campaign A'],
    'sales': [150, 200, 250, 350, 220, 280, 180],  # Sample sales figures for sandals
    'reviews_count': [12, 18, 28, 32, 20, 25, 15],  # Sample review counts for sandals
    'average_rating': [4.5, 4.6, 4.3, 4.8, 4.7, 4.4, 4.5]  # Sample average ratings for sandals
}

# Create a DataFrame
df_sales = pd.DataFrame(data)

# Analyze sandals sales data by campaign
results = df_sales.groupby('campaign').agg({
    'sales': 'sum',              # Total sales per campaign
    'reviews_count': 'sum',      # Total reviews per campaign
    'average_rating': 'mean'     # Average rating per campaign
}).reset_index()

# Display the results
print("A/B Testing Results for Sandals Sales:")
print(results)


A/B Testing Results for Sandals Sales:
     campaign  sales  reviews_count  average_rating
0  Campaign A    750             65           4.575
1  Campaign B    880             85           4.500


**Campaign Performance:**

* Campaign B significantly outperformed Campaign A in total sales and reviews, indicating it was a more effective marketing strategy.
Customer Engagement:

* The increase in reviews for Campaign B highlights higher customer interaction, which is essential for building brand loyalty and repeat purchases.
Customer Satisfaction:

* Campaign A had a better average rating, which may indicate better individual customer experiences, even with fewer sales. It is important to further investigate the feedback to understand the aspects that led to higher satisfaction.
Recommendations:

* Future Campaigns: Consider combining elements from both campaigns. For example, using a bundling offer with enhanced customer service or product satisfaction guarantees to boost both sales and ratings.

* Customer Feedback: Collect more detailed qualitative feedback from customers to understand their experiences and identify potential areas for improvement.
Ongoing Monitoring: Continue A/B testing with variations in promotions and pricing strategies to refine marketing approaches over time.

# **FUNNEL ANALYSIS**

In [13]:
import pandas as pd

# Load the data
df = pd.read_csv('adidas_usa.csv')

# Check the columns
print("Available columns in the DataFrame:")
print(df.columns)

# Assume we want to segment by product category and color, as well as sales data
# Create a new DataFrame focusing on relevant attributes
segmentation_data = df[['category', 'color', 'sku', 'selling_price', 'availability', 'average_rating', 'reviews_count']]

# Group by category and color, and calculate relevant metrics
segmented_data = (
    segmentation_data
    .groupby(['category', 'color'])
    .agg(
        total_sales=('sku', 'nunique'),  # Count unique SKUs sold
        average_price=('selling_price', 'mean'),  # Average selling price
        average_rating=('average_rating', 'mean'),  # Average rating
        total_reviews=('reviews_count', 'sum')  # Total reviews
    )
    .reset_index()
)

# Display the segmented data
print(segmented_data)

# Optional: Further segment based on average price or rating
# For instance, you could segment by price brackets
price_bins = [0, 50, 100, 150, 200, float('inf')]
price_labels = ['Under $50', '$50-$100', '$100-$150', '$150-$200', 'Above $200']

segmented_data['price_segment'] = pd.cut(segmented_data['average_price'], bins=price_bins, labels=price_labels)

# Display the segmented data with price segments
print(segmented_data)

# Analyzing segment characteristics
for category in segmented_data['category'].unique():
    print(f"\nSegment Analysis for Category: {category}")
    category_data = segmented_data[segmented_data['category'] == category]
    print(category_data[['color', 'total_sales', 'average_price', 'average_rating', 'total_reviews']])


Available columns in the DataFrame:
Index(['index', 'url', 'name', 'sku', 'selling_price', 'original_price',
       'currency', 'availability', 'color', 'category', 'source',
       'source_website', 'breadcrumbs', 'description', 'brand', 'images',
       'country', 'language', 'average_rating', 'reviews_count', 'crawled_at'],
      dtype='object')
       category       color  total_sales  average_price  average_rating  \
0   Accessories       Beige            1      14.000000        4.700000   
1   Accessories       Black           20      42.250000        4.835000   
2   Accessories        Blue            7      27.428571        4.757143   
3   Accessories       Brown            1     112.000000        5.000000   
4   Accessories    Burgundy            2      29.500000        4.800000   
5   Accessories       Green            5      20.400000        4.840000   
6   Accessories        Grey            9      18.333333        4.922222   
7   Accessories  Multicolor           15      20.


Certainly! Below is a structured report based on the segmented data you provided. This report summarizes insights from the analysis of the Adidas product categories, focusing on total sales, average price, average rating, total reviews, and price segments.

Adidas Product Segmentation Report
Overview
This report presents the analysis of Adidas products segmented by category and color. Key performance metrics include total sales, average price, average rating, and total reviews. The data reveals insights into customer preferences and product performance across different segments.

Segment Analysis by Category
1. Accessories Insights:

* The Black color is the best seller with 20 units sold and a high average rating of 4.84, indicating strong customer satisfaction.
* The Brown accessory has the highest average price at $112 but low total sales, suggesting it may be a niche item.
* The White accessories receive the highest total reviews, suggesting they are popular among customers.

2. Clothing Insights:

* Black clothing is the top performer with 78 units sold and significant customer engagement through reviews.
* Blue and Pink are also strong sellers with high review counts, indicating favorable customer reception.
* The Multi color option, while selling only 2 units, boasts an impressive average rating of 5.00.

3. Shoes Insights:

* White shoes dominate in sales with 166 units sold, supported by a vast number of reviews (157,486), indicating a strong market presence.
* Black and Blue shoes also perform well with 89 and 37 units sold, respectively.
* The Turquoise color, despite limited sales, has a high average price of $107.

Segmentation Summary Price
* Most items are priced under $50, which aligns with higher sales volumes in
the Accessories and Clothing categories.

* Items priced above $100, like some shoes, show lower sales numbers but could indicate premium positioning.


# **Conclusion**
The segmentation analysis provides valuable insights into Adidas' product performance across different categories and colors. The Accessories category demonstrates diverse color options with varying customer engagement, while Clothing and Shoes categories show clear leaders in sales and customer ratings.

Recommendations:
1. Focus on Best-Sellers: Enhance marketing efforts for top-selling items, particularly in the Shoes and Clothing categories.
2. Product Diversification: Consider expanding the product line for underperforming colors or categories based on customer preferences.
3. Price Strategy: Evaluate pricing strategies for high-demand segments to maximize revenue without sacrificing customer satisfaction.

In [15]:
import pandas as pd

# Sample Data: Create a DataFrame for demonstration
data = {
    'year': [2022, 2023],
    'sales': [10000, 12000]
}

# Load the dataset into a DataFrame
df = pd.DataFrame(data)

# Display the original DataFrame
print("Sales Data:")
print(df)

# Sort the DataFrame by 'year' to ensure correct order for growth calculation
df = df.sort_values(by='year')

# Calculate the YOY Growth
# Calculate percentage change and multiply by 100 to convert it to percentage
df['YOY Growth (%)'] = df['sales'].pct_change() * 100

# Display the DataFrame with YOY Growth
print("\nSales Data with YOY Growth:")
print(df)


Sales Data:
   year  sales
0  2022  10000
1  2023  12000

Sales Data with YOY Growth:
   year  sales  YOY Growth (%)
0  2022  10000             NaN
1  2023  12000            20.0


# **Interpretation**
* The sales increased from $10,000 in 2022 to $12,000 in 2023.
* The calculated YOY growth of 20.0% indicates a positive trend, suggesting that sales performance improved significantly in 2023 compared to 2022.

This growth can be attributed to various factors, including enhanced marketing strategies, improved product offerings, or increased demand for Adidas products during the period.

# **Conclusion**
* The analysis shows that Adidas experienced a strong sales increase of 20% from 2022 to 2023.
* This trend is encouraging and may provide insights for future business strategies.
Continuous monitoring of sales data will be essential to sustain growth and identify further opportunities.

# **Recommendations**
* Market Analysis: Conduct a deeper analysis to identify the key drivers of sales growth.
* Customer Feedback: Gather customer insights to understand what contributed to the increased sales and areas for improvement.
* Strategic Planning: Leverage the positive growth to inform future marketing and sales strategies.
